src/plugins/kibana/server/lib/schemas/resources/index_pattern_schema.js

```diff
@@ -6,6 +6,7 @@ module.exports = Joi.object({
   time_field_name: Joi.string(),
   interval_name: Joi.string(),
   not_expandable: Joi.boolean(),
+  fieldFilters: Joi.string().default('[]'),
```

Comment:
The mapping of the index pattern in https://github.com/elastic/kibana/blob/91b6d925292f7bb62f059f2ba06c41e775ef2eb7/src/ui/public/index_patterns/_index_pattern.js#L41 is camel case. Out of curiosity, is it normal ? Or will it change to reflect the snake_case format ?